lectures/linear_equations.md

```diff
@@ -297,7 +290,7 @@ $$
 \begin{bmatrix}
     -6 \\
     14
-\end{bmatrix}
+\end{bmatrix}.
```

Comment:
The previous ones look better to me. Still, let @jstac make the decision for equations involving matrix/vectors.